### ISSUE TITLE
Enable remote use

### ISSUE THREAD

**Issue by roaldarbol** [BeeHive-org/BeeHive#64]
Lately I've been thinking quite a it about how versatile the BeeHive could be even for remote field work, moving the applicability from just the lab out into the field. 

**Battery**
On obstacle is that it should be possible to power it with a battery. And then the actual implementation depends on whether we would still like for it to be able to pull 12V. Here's some ideas, but I'm sure there are more ways than I've thought of.

If battery life is of the essence, LiPo batteries would be ideal and have also been used for many similar things, although they can only deliver 5V as far as I know (e.g. the [PiJuice](https://uk.pi-supply.com/collections/power-supplies-and-batteries/products/pijuice-standard) - their implementation is quite cool, basically a hat - we could consider something similar, as long as the standoffs are like the BeeHive, it could stack up). Pimoroni uses a 2-pole JST PH battery connector for their batteries.

Of course there's a few ways to do this with normal AA batteries as well. There are some decent options on the market now (e.g. [rechargeable Energizers](https://www.amazon.co.uk/dp/B01DJ2QBEO?linkCode=gs4&tag=reactual-21)). Having a [10-battery holder ](https://thepihut.com/products/10aa-holder)can deliver 12V, but it would need a 2-pin screwgate (so adding one to the board could be a possible solution). There are AA holders that come with a barrel jack, but the ones I've seen (e.g. [this one](https://thepihut.com/products/6xaa-battery-holder-with-straight-dc-barrel-jack)) don't take as many batteries and can't provide 12V. 

**On/off switch**
Another thing that would then be needed is a power switch.

**Barrel jack/screwgate switch**
And another switch that might be good to have is a switch so you can choose where to pull the power from.

In summary, I'd suggest either:
1) The easy way forward: Add a 2-pin screwgate, a switch to choose barrel/screwgate, and an ON/OFF switch (AA battery solution).
2) The harder way: Add a 2-pin JST connector and ensure that 12V cannot be pulled from it - and again a switch.

**Comment by isobianin** [comment]
Thanks for the input @roaldarbol and well documented thoughts and ideas. Maybe a bit of a spoiler, but we have been working on a new design for the main hub:
<img src="https://github.com/BeeHive-org/BeeHive/blob/master/hardware/PCBs/central_hub_alt/beehive_PCB.png?raw=true">
In the top right corner you can see a 12V barreljack that can be turned on/off via a jumper wire. Hopefully, when this one is manufactured, it might almost be an answer to this issue.
@amchagas and I decided that it might be convenient to be able to turn on/off each individual 5V line in a similar way. Thus there are a lot of connectors on the new board.
